api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -24,7 +24,7 @@
  * <p>A Converter can specify a {@link javax.annotation.Priority}.
```

Comment:
By the way, CDI has dependencies on JSR-250 for its interceptors/decorators/alternatives. Since Config has dependencies on CDI, JSR-250 dependency may not need to be discussed.